hr_job_category/views/hr_view.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_job_form" model="ir.ui.view">
+        <field name="name">hr.job.category</field>
+        <field name="model">hr.job</field>
+        <field name="inherit_id" ref="hr.view_hr_job_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[hasclass('oe_title')]" position="inside">
+                <label for="category_ids" class="oe_edit_only" groups="base.group_hr_manager"/>
```

Comment:
Wait for https://github.com/OCA/hr/pull/548 to be merged or just cherry-pick it's commit to fix the issue mentioned in runbot